.github/CODEOWNERS

```diff
@@ -1,3 +1,16 @@
+################
+# Misc
+################
+/*                               @azure/azure-sdk-eng
+/.azure-pipelines/               @azure/azure-sdk-eng
+/.devcontainer/                  @azure/azure-sdk-eng
+/artifacts/                      @azure/azure-sdk-eng
+/config/                         @azure/azure-sdk-eng
+/doc/                            @azure/azure-sdk-eng
+/scripts/                        @azure/azure-sdk-eng
```

Comment:
This scripts folder should be moved under eng/scripts at some point.

Reply:
Created:
- #5056